cdap-ui/app/directives/epoch/charts.js

```diff
@@ -72,7 +72,7 @@ ngEpoch.controller('epochController', function ($scope, $compile, caskWindowMana
         }
         else if (newVal && newVal.length) {
           for (var i = 0; i < newVal.length; i++) {
-            $scope.me.push([newVal[i]]);
+            $scope.me.push(newVal[i]);
```

Comment:
I wrap it in an array here:
https://github.com/caskdata/cdap/pull/2035/files#diff-461271d65a10d30a3b9a8f79b05e0349R157

For multiple metrics, it's no longer a single element array, but instead an array of n elements, where n is the number of metrics being polled/displayed.